packages/frontend/scss/message/_message.scss

```diff
@@ -470,8 +470,21 @@
   }
 
   img.quoted-image {
+    // Remember that width and height need to be set prior to the image
+    // getting actually loaded by the browser, to avoid content shifts.
+    // Same goes for messages' `.attachment-content`.
+    // See https://github.com/deltachat/deltachat-desktop/issues/4404.
+    //
+    // Setting just the height is not enough, because there might be text
+    // displayed beside the image, which might wrap and thus change height.
+    //
+    // In addition, really short and wide images could make the message
+    // overflow the chat container, for non-self-sent messages at least.
     height: var(--quote-img-size);
-    // do not set width, to keep the aspect ratio (see https://github.com/deltachat/deltachat-desktop/issues/3931)
+    width: var(--quote-img-size);
```

Comment:
Is object-fit: scale-down in conjunction with width: 100% not enough (and better since it does not force any image to have a 36px width)?

I tested various variants with extreme landscape ratios and text and it looked better.

But I can also live with the current solutions, since we are going to find  amore adaptive generic solution for images anyway (hopefully)

Reply:
Have you tried quoting a in image with a captoion or just an image?

Reply:
I tried with text and without. But it's true, with an extrem ratio (like 1300 x 30 px) it hides the text. So let's solve that hopefully soon as said above

Reply:
`width: 100%` simply makes it take up 50% of the flex container, which can be adjusted with `flex-shrink`. IDK if this is what we want.